src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -798,15 +798,20 @@ public void UpdateReportTabStatistics(Patch patch)
                 var dataset = speciesPopulationChart.GetDataSet(entry.Key.FormattedName);
 
                 var extinctInPatch = entry.Value <= 0;
+                var extinctEverywhere = snapshot.RecordedSpeciesInfo[entry.Key].Population <= 0;
 
                 // Clamp population number so it doesn't go into the negatives
                 var population = extinctInPatch ? 0 : entry.Value;
 
                 var dataPoint = new DataPoint
                 {
                     Value = new Vector2((float)snapshot.TimePeriod, population),
-                    Size = extinctInPatch ? 12 : 7,
-                    IconType = extinctInPatch ? DataPoint.MarkerIcon.Cross : DataPoint.MarkerIcon.Circle,
+                    Size = extinctInPatch ? extinctEverywhere ? 28 : 12 : 7,
+                    IconType = extinctInPatch ?
+                        extinctEverywhere ?
```

Comment:
I feel like nested ternary operators are a bad code smell, and this should be refactored out to be an if-block at this point for readability.

Reply:
I usually never use ternary ops. But here I really felt they were intuitive, especially when indented, as you can grasp the whole structure in an eyeblick due to the short code length. That's why I kept the nested structure

However, I'm not against going with if-blocks, so if you feel ternary operators may hamper readability, I'lll just use if-blocks instead.

Reply:
I have a personal rule that nested ternaries are a no-go. non-nested ternaries are fine to shorten code, but here I think the readability is actually much worse due to the nested ternary usage, so I would not use that here.

Reply:
I'm fine with having this rule. Could be worth to mention it in the style guide and add it to the style cop though.

Reply:
I would have enabled a check if there was one (feel free to enable it if I happened to miss one).
https://github.com/Revolutionary-Games/Thrive/issues/2454

Reply:
I don't know about the checks, I just know it wanted to format my ternary op nesting. I'll take a look during the week-end.

Reply:
Only thing I could find in Rider settings is enforcing a different style for nested ternaries, which will make them more noticeable (and harder to manually write right).